inventory/00-static/ocp-infra.yaml

```diff
@@ -12,5 +12,17 @@ all:
                 vendor: lenovo
               hosts:
                 ctl-0.nerc-ocp-infra.rc.fas.harvard.edu:
+                  row: 07
+                  pod: C
+                  cage: 03
+                  unitnumber: 09-10
```

Comment:
I would prefer `unitnumber` to be a single value (typically, the lowest position the device occupies). If you want to record the device height, lets add a `height` parameter. Also, be careful writing numbers with leading zeroes: I don't think it's a problem here, but some tools interpret a leading 0 to mean "this is an octal number". If these are supposed to be numeric, strip the leading zero.

Reply:
Will do.